iris-GmbH/meta-iris-base
# [RDPHOEN-1165] fitimage for netboot

Fix nfsboot.
## How to test
Build any uuu image
```
bitbake mc:imx8mp-evk:irma6-base-uuu
```

Flash generated flashalll.uuu. Unpack `rootfs ... tar.bz2` to your nfs location.

Build netboot fitimage
```
bitbake mc:imx8mp-evk:irma6-fitimage-netboot
```

Copy `irma6-fitimage-netboot.itb.signed` to your tftp location, rename to `fitImage.signed`.

Boot board, check if mmcboot is working. Reboot to U-Boot console. `setenv nfsroot` to your nfs location. Run nfsboot with
```
run netfitboot
```

If you don't have a dhcp server running, also `setenv serverip` and replace `dhcp` with `tftp` in the `netfitboot` variable.
